specification/eventgrid/resource-manager/Microsoft.EventGrid/2017-09-15-preview/EventGrid.json

```diff
@@ -1015,6 +1015,68 @@
                     }
                 },
                 "x-ms-long-running-operation": true
+            },
+            "patch": {
+                "tags": [
+                    "Topics"
+                ],
+                "summary": "Update a topic",
+                "description": "Asynchronously updates a topic with the specified parameters.",
+                "operationId": "Topics_Update",
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "name": "resourceGroupName",
+                        "in": "path",
+                        "description": "The name of the resource group within the user's subscription.",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "topicName",
+                        "in": "path",
+                        "description": "Name of the topic",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "topicUpdateParameters",
+                        "in": "body",
+                        "description": "Topic update information",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/TopicUpdateParameters"
+                        }
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
```

Comment:
Is there a chance that the service returns `200` status code? if yes the generated client might consider it to be an error response.

Reply:
Good point. No, the service always handles this asynchronously and always returns only 201, so I think we are fine leaving it with just 201 status code. Please let me know if that is fine.